src/view-utilities.js

```diff
@@ -124,20 +125,20 @@ var viewUtilities = function (cy, options) {
 
 
   //Zoom selected Nodes
-  instance.zoom = function ( eles){
+  instance.zoomToSelected = function (eles, zoomSpeed){
     eles.unselect();
     cy.animate({
       fit: {
         eles: eles,
         padding: 20
       }
     }, {
-      duration: 1000
+      duration: 30 * zoomSpeed //Default:1500( %50)
```

Comment:
Please add an option named as "zoomAnimationDuration" to the options object at the beginning of the index.js file, after setDisplayOnHide option, and set its value to 1500. Then call that option here as 
`duration: options.zoomAnimationDuration`
Remove zoomSpeed parameter.